prac_08/box_layout_demo.py

```diff
@@ -0,0 +1,20 @@
+from kivy.app import App
+from kivy.lang import Builder
+
+
+class BoxLayoutDemo(App):
+    def build(self):
```

Comment:
You may to have forgotten to add your docstrings only box_layout_demo has them missing.